INFO     [🔧TEST CONFIG]:conftest.py:87 Loading test configuration
INFO     [🔧TEST CONFIG]:conftest.py:378 Navigating to: https://console.dev.apolo.us/
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/config
Request Body:
<not available>
Response Body (first 1000 chars):
{"authorized": false, "auth_url": "https://auth.dev.apolo.us/authorize", "token_url": "https://auth.dev.apolo.us/oauth/token", "logout_url": "https://auth.dev.apolo.us/v2/logout?returnTo=https://console.dev.apolo.us", "client_id": "q3I0OzzGnTDkhRmpeJ7WWgaTCucmVxTL", "audience": "https://api.dev.apolo.us", "callback_urls": ["http://127.0.0.1:54540", "http://127.0.0.1:54541", "http://127.0.0.1:54542"], "headless_callback_url": "https://api.dev.apolo.us/oauth/show-code", "success_redirect_url": "https://api.dev.apolo.us/oauth/login"}
INFO     [🔧TEST CONFIG]:conftest.py:93 Creating data manager
INFO     [🔧TEST CONFIG]:conftest.py:123 Creating users manager
INFO     [🔧TEST CONFIG]:conftest.py:109 Creating API helper
INFO     [🔧TEST CONFIG]:conftest.py:117 Creating Apolo CLI instance
INFO     [🔧TEST CONFIG]:conftest.py:205 Continue using UserData(email='regression-ceadr7jsak@apolo.us', username='regression-ceadr7jsak', password='TM4wn6ACLjm&') as main test user...
INFO     [👤UsersManager]:users_manager.py:73 Setting main user to UserData(email='regression-ceadr7jsak@apolo.us', username='regression-ceadr7jsak', password='TM4wn6ACLjm&')
INFO     [🔧TEST CONFIG]:conftest.py:235 Continue using UserData(email='regression-nl69exwsvz@apolo.us', username='regression-nl69exwsvz', password='8ISTI6#9!ZOs') as second test user...
INFO     [👤UsersManager]:users_manager.py:82 Setting second user to UserData(email='regression-nl69exwsvz@apolo.us', username='regression-nl69exwsvz', password='8ISTI6#9!ZOs')
INFO     [🔧TEST CONFIG]:conftest.py:240 Continue using UserData(email='regression-p7oso6fqhi@apolo.us', username='regression-p7oso6fqhi', password='ICnthK!X6TGG') as third test user...
INFO     [👤UsersManager]:users_manager.py:91 Setting third user to UserData(email='regression-p7oso6fqhi@apolo.us', username='regression-p7oso6fqhi', password='ICnthK!X6TGG')
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Verify Apolo CLI client is installed
INFO     [🖥apolo_CLI]:apolo_runner.py:25 check CLI version. Running command via cli:

apolo --version

INFO     [🖥apolo_CLI]:apolo_runner.py:86 check CLI version succeeded:

Apolo Platform Client 25.8.1

INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Verify Apolo CLI client is installed
INFO     [📘TEST_INFO]:reporting.py:141 ============================================================
INFO     [📘TEST_INFO]:reporting.py:142 📁 SUITE STARTED: CLI Secrets
INFO     [📘TEST_INFO]:reporting.py:143 ============================================================
INFO     [📘TEST_INFO]:reporting.py:86 ------------------------------------------------------------
INFO     [📘TEST_INFO]:reporting.py:87 📝 TEST started: Admin create secret with no project created via CLI
INFO     [📘TEST_INFO]:reporting.py:88 ------------------------------------------------------------
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Login via UI
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Click login button
INFO     [🎭 Playwright]:base_page.py:16 <AuthPage>: Click log in button
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Click login button
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_click_login_button_success_17f26b.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Enter email
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Enter regression-ceadr7jsak@apolo.us email
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Enter email
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_enter_email_success_4b73f9.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Enter password
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Enter ********* password
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Enter password
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_enter_password_success_3bc82e.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Click Continue button
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Click continue button
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Waiting for spinner
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 500] GET https://api.dev.apolo.us/api/v1/config
Request Body:
<not available>
Response Body (first 1000 chars):
{"error": "Unexpected exception ClientResponseError: 500, message='Internal Server Error', url='http://platform-admin:8080/apis/admin/v1/users/regression-ceadr7jsak?include=orgs&include=clusters&include=projects'. Path with query: /api/v1/config."}
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/users/regression-ceadr7jsak/permissions?uri=blob:
Request Body:
<not available>
Response Body (first 1000 chars):
[{"uri": "blob://default/regression-org-x2ill7bl19qh", "action": "manage"}]
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Timeout error for spinner
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Click Continue button
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_click_continue_button_success_7deffe.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Verify that Welcome new user page displayed
INFO     [🎭 Playwright]:base_page.py:16 <WelcomeNewUserPage>: Check if page loaded
ERROR    [📘TEST_INFO]:reporting.py:56 ❌ STEP failed: Verify that Welcome new user page displayed
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (fail): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/verify_ui_page_displayed_fail_ebda91.png
ERROR    [📘TEST_INFO]:reporting.py:56 ❌ STEP failed: Login via UI
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (fail): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_login_fail_3b903d.png
ERROR    [📘TEST_INFO]:reporting.py:95 ❌ TEST failed: Admin create secret with no project created via CLI
Exception: [AssertionError] during step: Login via UI
Message: [AssertionError] during step: Verify that Welcome new user page displayed
Message: Locator expected to be visible
Actual value: <element(s) not found> 
Call log:
  - Expect "to_be_visible" with timeout 5000ms
  - waiting for locator("h3.truncate.text-h3").filter(has_text="Welcome, regression-ceadr7jsak@apolo.us")

INFO     [📘TEST_INFO]:reporting.py:98 ------------------------------------------------------------
INFO     [🌐API_helper]:api_helper.py:54 GET https://api.dev.apolo.us/apis/admin/v1/orgs
INFO     [🌐API_helper]:api_helper.py:165 Status: 200. Response: []
INFO     [🔧TEST CONFIG]:conftest.py:291 Cleaning up 0 organisations
INFO     [🔧TEST CONFIG]:conftest.py:321 Closing all browser sessions
INFO     [🔧TEST CONFIG]:conftest.py:327 Stopped Playwright
WARNING  [🔧TEST CONFIG]:conftest.py:337 Failed to close context: BrowserContext.close: Target page, context or browser has been closed
INFO     [🔧TEST CONFIG]:conftest.py:342 Closed browser (post-stop)
INFO     [🔧TEST CONFIG]:conftest.py:347 Browser cleanup finished
INFO     [🔧TEST CONFIG]:conftest.py:87 Loading test configuration
INFO     [🔧TEST CONFIG]:conftest.py:378 Navigating to: https://console.dev.apolo.us/
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/config
Request Body:
<not available>
Response Body (first 1000 chars):
{"authorized": false, "auth_url": "https://auth.dev.apolo.us/authorize", "token_url": "https://auth.dev.apolo.us/oauth/token", "logout_url": "https://auth.dev.apolo.us/v2/logout?returnTo=https://console.dev.apolo.us", "client_id": "q3I0OzzGnTDkhRmpeJ7WWgaTCucmVxTL", "audience": "https://api.dev.apolo.us", "callback_urls": ["http://127.0.0.1:54540", "http://127.0.0.1:54541", "http://127.0.0.1:54542"], "headless_callback_url": "https://api.dev.apolo.us/oauth/show-code", "success_redirect_url": "https://api.dev.apolo.us/oauth/login"}
INFO     [🔧TEST CONFIG]:conftest.py:93 Creating data manager
INFO     [🔧TEST CONFIG]:conftest.py:123 Creating users manager
INFO     [🔧TEST CONFIG]:conftest.py:109 Creating API helper
INFO     [🔧TEST CONFIG]:conftest.py:117 Creating Apolo CLI instance
INFO     [🔧TEST CONFIG]:conftest.py:205 Continue using UserData(email='regression-ceadr7jsak@apolo.us', username='regression-ceadr7jsak', password='TM4wn6ACLjm&') as main test user...
INFO     [👤UsersManager]:users_manager.py:73 Setting main user to UserData(email='regression-ceadr7jsak@apolo.us', username='regression-ceadr7jsak', password='TM4wn6ACLjm&')
INFO     [🔧TEST CONFIG]:conftest.py:235 Continue using UserData(email='regression-nl69exwsvz@apolo.us', username='regression-nl69exwsvz', password='8ISTI6#9!ZOs') as second test user...
INFO     [👤UsersManager]:users_manager.py:82 Setting second user to UserData(email='regression-nl69exwsvz@apolo.us', username='regression-nl69exwsvz', password='8ISTI6#9!ZOs')
INFO     [🔧TEST CONFIG]:conftest.py:240 Continue using UserData(email='regression-p7oso6fqhi@apolo.us', username='regression-p7oso6fqhi', password='ICnthK!X6TGG') as third test user...
INFO     [👤UsersManager]:users_manager.py:91 Setting third user to UserData(email='regression-p7oso6fqhi@apolo.us', username='regression-p7oso6fqhi', password='ICnthK!X6TGG')
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Verify Apolo CLI client is installed
INFO     [🖥apolo_CLI]:apolo_runner.py:25 check CLI version. Running command via cli:

apolo --version

INFO     [🖥apolo_CLI]:apolo_runner.py:86 check CLI version succeeded:

Apolo Platform Client 25.8.1

INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Verify Apolo CLI client is installed
INFO     [📘TEST_INFO]:reporting.py:86 ------------------------------------------------------------
INFO     [📘TEST_INFO]:reporting.py:87 📝 TEST started: Admin create secret with no project created via CLI
INFO     [📘TEST_INFO]:reporting.py:88 ------------------------------------------------------------
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Login via UI
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Click login button
INFO     [🎭 Playwright]:base_page.py:16 <AuthPage>: Click log in button
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Click login button
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_click_login_button_success_c70d6c.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Enter email
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Enter regression-ceadr7jsak@apolo.us email
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Enter email
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_enter_email_success_91532d.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Enter password
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Enter ********* password
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Enter password
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_enter_password_success_4324b7.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Click Continue button
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Click continue button
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Waiting for spinner
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 500] GET https://api.dev.apolo.us/api/v1/config
Request Body:
<not available>
Response Body (first 1000 chars):
{"error": "Unexpected exception ClientResponseError: 500, message='Internal Server Error', url='http://platform-admin:8080/apis/admin/v1/users/regression-ceadr7jsak?include=orgs&include=clusters&include=projects'. Path with query: /api/v1/config."}
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/users/regression-ceadr7jsak/permissions?uri=blob:
Request Body:
<not available>
Response Body (first 1000 chars):
[{"uri": "blob://default/regression-org-x2ill7bl19qh", "action": "manage"}]
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Timeout error for spinner
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Click Continue button
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_click_continue_button_success_49ff98.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Verify that Welcome new user page displayed
INFO     [🎭 Playwright]:base_page.py:16 <WelcomeNewUserPage>: Check if page loaded
ERROR    [📘TEST_INFO]:reporting.py:56 ❌ STEP failed: Verify that Welcome new user page displayed
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (fail): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/verify_ui_page_displayed_fail_245983.png
ERROR    [📘TEST_INFO]:reporting.py:56 ❌ STEP failed: Login via UI
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (fail): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_login_fail_9bcdb6.png
ERROR    [📘TEST_INFO]:reporting.py:95 ❌ TEST failed: Admin create secret with no project created via CLI
Exception: [AssertionError] during step: Login via UI
Message: [AssertionError] during step: Verify that Welcome new user page displayed
Message: Locator expected to be visible
Actual value: <element(s) not found> 
Call log:
  - Expect "to_be_visible" with timeout 5000ms
  - waiting for locator("h3.truncate.text-h3").filter(has_text="Welcome, regression-ceadr7jsak@apolo.us")

INFO     [📘TEST_INFO]:reporting.py:98 ------------------------------------------------------------
INFO     [🌐API_helper]:api_helper.py:54 GET https://api.dev.apolo.us/apis/admin/v1/orgs
INFO     [🌐API_helper]:api_helper.py:165 Status: 200. Response: [{'name': 'regression-org-x2ill7bl19qh', 'balance': {'credits': '500', 'spent_credits': '0'}, 'user_default_credits': None, 'notification_intervals': {'balance_projection_seconds': [604800, 259200, 86400], 'balance_amount': [0], 'balance_depletion_seconds': [86400, 604800]}}]
INFO     [🔧TEST CONFIG]:conftest.py:291 Cleaning up 1 organisations
INFO     [🌐API_helper]:api_helper.py:172 Delete organization response: <ClientResponse(https://api.dev.apolo.us/apis/admin/v1/orgs/regression-org-x2ill7bl19qh) [200 OK]>
<CIMultiDictProxy('Access-Control-Allow-Credentials': 'true', 'Content-Length': '274', 'Content-Type': 'application/json; charset=utf-8', 'Date': 'Tue, 16 Sep 2025 03:16:53 GMT', 'Server': 'Python/3.13 aiohttp/3.12.15', 'Vary': 'Origin', 'x-service-version': 'platform-admin/0.0.0.post1.dev0+4cdab15', 'Via': '1.1 google', 'Alt-Svc': 'h3=":443"; ma=2592000,h3-29=":443"; ma=2592000')>

INFO     [🔧TEST CONFIG]:conftest.py:316 Removed organisation: regression-org-x2ill7bl19qh
INFO     [🔧TEST CONFIG]:conftest.py:321 Closing all browser sessions
INFO     [🔧TEST CONFIG]:conftest.py:327 Stopped Playwright
WARNING  [🔧TEST CONFIG]:conftest.py:337 Failed to close context: BrowserContext.close: Target page, context or browser has been closed
INFO     [🔧TEST CONFIG]:conftest.py:342 Closed browser (post-stop)
INFO     [🔧TEST CONFIG]:conftest.py:347 Browser cleanup finished
INFO     [🔧TEST CONFIG]:conftest.py:87 Loading test configuration
INFO     [🔧TEST CONFIG]:conftest.py:378 Navigating to: https://console.dev.apolo.us/
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/config
Request Body:
<not available>
Response Body (first 1000 chars):
{"authorized": false, "auth_url": "https://auth.dev.apolo.us/authorize", "token_url": "https://auth.dev.apolo.us/oauth/token", "logout_url": "https://auth.dev.apolo.us/v2/logout?returnTo=https://console.dev.apolo.us", "client_id": "q3I0OzzGnTDkhRmpeJ7WWgaTCucmVxTL", "audience": "https://api.dev.apolo.us", "callback_urls": ["http://127.0.0.1:54540", "http://127.0.0.1:54541", "http://127.0.0.1:54542"], "headless_callback_url": "https://api.dev.apolo.us/oauth/show-code", "success_redirect_url": "https://api.dev.apolo.us/oauth/login"}
INFO     [🔧TEST CONFIG]:conftest.py:93 Creating data manager
INFO     [🔧TEST CONFIG]:conftest.py:123 Creating users manager
INFO     [🔧TEST CONFIG]:conftest.py:109 Creating API helper
INFO     [🔧TEST CONFIG]:conftest.py:117 Creating Apolo CLI instance
INFO     [🔧TEST CONFIG]:conftest.py:205 Continue using UserData(email='regression-ceadr7jsak@apolo.us', username='regression-ceadr7jsak', password='TM4wn6ACLjm&') as main test user...
INFO     [👤UsersManager]:users_manager.py:73 Setting main user to UserData(email='regression-ceadr7jsak@apolo.us', username='regression-ceadr7jsak', password='TM4wn6ACLjm&')
INFO     [🔧TEST CONFIG]:conftest.py:235 Continue using UserData(email='regression-nl69exwsvz@apolo.us', username='regression-nl69exwsvz', password='8ISTI6#9!ZOs') as second test user...
INFO     [👤UsersManager]:users_manager.py:82 Setting second user to UserData(email='regression-nl69exwsvz@apolo.us', username='regression-nl69exwsvz', password='8ISTI6#9!ZOs')
INFO     [🔧TEST CONFIG]:conftest.py:240 Continue using UserData(email='regression-p7oso6fqhi@apolo.us', username='regression-p7oso6fqhi', password='ICnthK!X6TGG') as third test user...
INFO     [👤UsersManager]:users_manager.py:91 Setting third user to UserData(email='regression-p7oso6fqhi@apolo.us', username='regression-p7oso6fqhi', password='ICnthK!X6TGG')
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Verify Apolo CLI client is installed
INFO     [🖥apolo_CLI]:apolo_runner.py:25 check CLI version. Running command via cli:

apolo --version

INFO     [🖥apolo_CLI]:apolo_runner.py:86 check CLI version succeeded:

Apolo Platform Client 25.8.1

INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Verify Apolo CLI client is installed
INFO     [📘TEST_INFO]:reporting.py:86 ------------------------------------------------------------
INFO     [📘TEST_INFO]:reporting.py:87 📝 TEST started: Admin create secret with no project created via CLI
INFO     [📘TEST_INFO]:reporting.py:88 ------------------------------------------------------------
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Login via UI
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Click login button
INFO     [🎭 Playwright]:base_page.py:16 <AuthPage>: Click log in button
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Click login button
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_click_login_button_success_59379a.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Enter email
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Enter regression-ceadr7jsak@apolo.us email
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Enter email
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_enter_email_success_992bb1.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Enter password
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Enter ********* password
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Enter password
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_enter_password_success_d7345a.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Click Continue button
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Click continue button
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Waiting for spinner
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/config
Request Body:
<not available>
Response Body (first 1000 chars):
{"authorized": true, "orgs": [], "clusters": [], "projects": [], "admin_url": "https://api.dev.apolo.us/apis/admin/v1", "auth_url": "https://auth.dev.apolo.us/authorize", "token_url": "https://auth.dev.apolo.us/oauth/token", "logout_url": "https://auth.dev.apolo.us/v2/logout?returnTo=https://console.dev.apolo.us", "client_id": "q3I0OzzGnTDkhRmpeJ7WWgaTCucmVxTL", "audience": "https://api.dev.apolo.us", "callback_urls": ["http://127.0.0.1:54540", "http://127.0.0.1:54541", "http://127.0.0.1:54542"], "headless_callback_url": "https://api.dev.apolo.us/oauth/show-code", "success_redirect_url": "https://api.dev.apolo.us/oauth/login"}
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Waiting for spinner to disappear
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/users/regression-ceadr7jsak/permissions?uri=blob:
Request Body:
<not available>
Response Body (first 1000 chars):
[]
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 500] GET https://api.dev.apolo.us/apis/admin/v1/invites/incoming
Request Body:
<not available>
Response Body (first 1000 chars):
{"error": "Unexpected exception: (sqlalchemy.dialects.postgresql.asyncpg.ProgrammingError) <class 'asyncpg.exceptions.UndefinedColumnError'>: column users.email does not exist\n[SQL: SELECT users.name, users.metadata, users.email, users.version \nFROM users \nWHERE users.name = %s]\n[parameters: ('regression-ceadr7jsak',)]\n(Background on this error at: https://sqlalche.me/e/14/f405). Path with query: /apis/admin/v1/invites/incoming."}
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/apis/admin/v1/users/regression-ceadr7jsak?include=clusters&include=system_role
Request Body:
<not available>
Response Body (first 1000 chars):
{"name": "regression-ceadr7jsak", "email": "regression-ceadr7jsak@apolo.us", "first_name": null, "last_name": null, "created_at": "2025-09-16T01:32:44.055799+00:00", "terms_accepted": true, "clusters": []}
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 500] GET https://api.dev.apolo.us/apis/admin/v1/users/regression-ceadr7jsak?include=orgs
Request Body:
<not available>
Response Body (first 1000 chars):
{"error": "Unexpected exception: (sqlalchemy.dialects.postgresql.asyncpg.ProgrammingError) <class 'asyncpg.exceptions.UndefinedColumnError'>: column users.email does not exist\n[SQL: SELECT users.name, users.metadata, users.email, users.version \nFROM users \nWHERE users.name = %s]\n[parameters: ('regression-ceadr7jsak',)]\n(Background on this error at: https://sqlalche.me/e/14/f405). Path with query: /apis/admin/v1/users/regression-ceadr7jsak?include=orgs."}
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Click Continue button
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_click_continue_button_success_4bafbc.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Verify that Welcome new user page displayed
INFO     [🎭 Playwright]:base_page.py:16 <WelcomeNewUserPage>: Check if page loaded
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Verify that Welcome new user page displayed
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/verify_ui_page_displayed_success_c0bb2d.png
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Login via UI
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_login_success_1a5a5e.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Add organization via API and reload page
INFO     [🔧DATA MANAGER]:test_data.py:56 Added organization: org_name=regression-org-t15lvz8ip9p4, gherkin_name=My-organization
INFO     [🌐API_helper]:api_helper.py:86 POST https://api.dev.apolo.us/apis/admin/v1/orgs?skip_auto_add_to_clusters=false with data: {'name': 'regression-org-t15lvz8ip9p4'}
INFO     [🌐API_helper]:api_helper.py:190 Add org regression-org-t15lvz8ip9p4 response: {'error': "Unexpected exception: (sqlalchemy.dialects.postgresql.asyncpg.ProgrammingError) <class 'asyncpg.exceptions.UndefinedColumnError'>: column users.email does not exist\n[SQL: SELECT users.name, users.metadata, users.email, users.version \nFROM users \nWHERE users.name = %s]\n[parameters: ('regression-org-t15lvz8ip9p4',)]\n(Background on this error at: https://sqlalche.me/e/14/f405). Path with query: /apis/admin/v1/orgs?skip_auto_add_to_clusters=false."}
ERROR    [📘TEST_INFO]:reporting.py:56 ❌ STEP failed: Add organization via API and reload page
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (fail): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_add_org_api_fail_89f767.png
ERROR    [📘TEST_INFO]:reporting.py:95 ❌ TEST failed: Admin create secret with no project created via CLI
Exception: [AssertionError] during step: Add organization via API and reload page
Message: Expected HTTP 201 response but got 500!
INFO     [📘TEST_INFO]:reporting.py:98 ------------------------------------------------------------
INFO     [🌐API_helper]:api_helper.py:54 GET https://api.dev.apolo.us/apis/admin/v1/orgs
INFO     [🌐API_helper]:api_helper.py:165 Status: 200. Response: []
INFO     [🔧TEST CONFIG]:conftest.py:291 Cleaning up 0 organisations
INFO     [🔧TEST CONFIG]:conftest.py:321 Closing all browser sessions
INFO     [🔧TEST CONFIG]:conftest.py:327 Stopped Playwright
WARNING  [🔧TEST CONFIG]:conftest.py:337 Failed to close context: BrowserContext.close: Target page, context or browser has been closed
INFO     [🔧TEST CONFIG]:conftest.py:342 Closed browser (post-stop)
INFO     [🔧TEST CONFIG]:conftest.py:347 Browser cleanup finished